cfme/infrastructure/config_management.py

```diff
@@ -442,7 +442,10 @@ class MgrAll(CFMENavigateStep):
 
     def step(self):
         from cfme.web_ui.menu import nav
-        nav._nav_to_fn('Configuration', 'Configuration Management')(None)
+        if self.obj.appliance.version > '5.7.0.8':
+            nav._nav_to_fn('Configuration', 'Configuration')(None)
```

Comment:
@psav, If I am not mistaken, this line should be changed to:

`
nav._nav_to_fn('Configuration', 'Management')(None)
`

![screenshot-20161102 112717](https://cloud.githubusercontent.com/assets/11808967/19925224/60095776-a0ef-11e6-9cdb-eaf786df96a8.png)

I did not test the code, but it looks good to me.